src/hooks/useGallery/README.md

```diff
@@ -0,0 +1,51 @@
+## useGallery
+
+The hook for opening the gallery
+
+### PropTypes
+
+_GalleryContextProvider_:
+
+| Property     | Type                          | Required | Values | Default | Description         |
+| :----------- | :---------------------------- | :------- | :----- | :------ | :------------------ |
+| theme        | `ThemeProviderProps['theme']` |          |        | `dark`  | The gallery theme   |
```

Comment:
`dark` is no longer default, let's keep it blank, since it's inherited from the root ThemeProvider